roles/kubernetes-apps/persistent_volumes/meta/main.yml

```diff
@@ -6,3 +6,10 @@ dependencies:
       - cloud_provider in [ 'openstack' ]
     tags:
       - persistent_volumes_openstack
+
+  - role: kubernetes-apps/persistent_volumes/aws
+    when:
+      - cloud_provider is defined
+      - cloud_provider in [ 'aws' ]
```

Comment:
Add check that aws_storage_classes is not empty and change lookup to match 'aws'
```
    when:
      - cloud_provider is defined
      - cloud_provider == 'aws'
      - aws_persistent_volumes_enabled
      - aws_storage_classes
```